frontend/ui/network/manager.lua

```diff
@@ -119,6 +119,20 @@ function NetworkMgr:getProxyMenuTable()
     }
 end
 
+function NetworkMgr:getRestoreMenuTable()
+    return {
+        text = _("Auto restore wifi after wake-up"),
```

Comment:
Yes, I have added a check for this menu option, now this menu option enabled for kobo only.